src/lib.rs

```diff
@@ -13,6 +13,7 @@
 
 #![crate_type = "lib"]
 #![cfg_attr(test, feature(test))]
+#![feature(proc_macro)]
```

Comment:
Why we need to enable it?

Reply:
Because our "make_static_metric" is implemented using proc_macro feature, so to use it we need to enable this feature.